cmd/notary/main_test.go

```diff
@@ -103,6 +103,7 @@ var exampleValidCommands = []string{
 	"status repo",
 	"publish repo",
 	"add repo v1 somefile",
+	"addhash repo targetv1 --sha256 aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa 10",
```

Comment:
Yeah I agree. I was thinking about this more after commenting as this method will reduce typos.